RELEASE_NOTES.md

```diff
@@ -20,6 +20,9 @@
 
 - The `Fuse` class has been moved to the `frequenz.sdk.timeseries` module.
 
+- The `ComponentGraph.components()` parameters `component_id` and `category_id`
+  were renamed to `component_category` and `component_categories`, respectively.
```

Comment:
You still kept the `\n` :cry:

Reply:
Well, actually you also forgot to change one `category_id` :scream:

Reply:
Actually you messed up completely... :rofl:

Reply:
yes 😆  but to be fair your original change request also suggested something wrong :P